src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color $transition-duration;
+
+    &:hover {
+      color: #00acdc;
```

Comment:
you should use variables for colors

Reply:
I thought about it but the checklist says "But don't overuse them, don't create variables for the value that's used just once". So I used it once and didn't create for in that case. But I really appreciate your attention to my work. Thank you,  Yaroslava ;)

Reply:
You use it twice actually)